pkg/features/features.go

```diff
@@ -41,21 +41,26 @@ const (
 	// blockdevice UUID algorithm mentioned in
 	// https://github.com/openebs/openebs/pull/2666
 	GPTBasedUUID Feature = "GPTBasedUUID"
+
 	// APIService feature flag starts the GRPC server which provides functionality to manage block devices
 	APIService Feature = "APIService"
+
+	UseOSDisk Feature = "UseOSDisk"
 )
 
 // supportedFeatures is the list of supported features. This is used while parsing the
 // feature flag given via command line
 var supportedFeatures = []Feature{
 	GPTBasedUUID,
 	APIService,
+	UseOSDisk,
 }
 
 // defaultFeatureGates is the default features that will be applied to the application
 var defaultFeatureGates = map[Feature]bool{
 	GPTBasedUUID: false,
 	APIService:   false,
+	UseOSDisk:    false,
```

Comment:
Should the GPTBasedUUID be `true` by default?

Reply:
yes, @kmova . I have already made those changes in another branch. Dont want to mix it up with current PR. Currently we set it to default by using the flag by default while starting. Setting this to true will help us to avoid that.  In the other branch, I also have fixes for some tests once this is made default.